pkg/operator/operator.go

```diff
@@ -33,6 +33,10 @@ const (
 	ownedManifestsDir = "owned-manifests"
 )
 
+func init() {
+	osconfigv1.AddToScheme(scheme.Scheme)
```

Comment:
Right package, but I still don't think this belongs in `init()`. Why not just call this in `New()`?

Reply:
> init() is called only once, operator.New can be called multiple times. Imho, AddToSheme will fail if the same type is registered twice.

I really don't like putting stuff in `init()` here personally. We could set up a new scheme to add these types to and pass that when creating the event recorder, right? That would solve the problem of being called more than once.

Reply:
Moved under `New()`.